src/node_mgmt/commands.py

```diff
@@ -211,6 +218,71 @@ def readall(s):
     return result
 
 
+def holykell_sensor_address_7(node):
+    return _change_address_holykell(target_slave_id=7)
+
+
+def holykell_sensor_address_8(node):
+    return _change_address_holykell(target_slave_id=8)
+
+
+def _change_address_holykell(target_slave_id: int) -> dict:
```

Comment:
I think it might be "cleaner" to also have the original slave ID as an input to the function. In this case it defaults to 1 (and probably always will), but what made me a little uncomfortable is the fact that we have a lot of hard-coded "1"s sprinkled through the code (lines 230, 238, 240, 264). So if we just have that as a variable it may make the functionality a bit more obvious

Reply:
Definitely, thanks I'm adding that